CONTRIBUTING.adoc

```diff
@@ -362,6 +362,28 @@ Read these blog posts for more good tips:
 - https://mtlynch.io/human-code-reviews-1/
 - https://mtlynch.io/human-code-reviews-2/
 
+== External contributors
+
+The Plutus team welcomes contributions from external contributors.
+However, this can easily become time-consuming and frustrating for both parties.
+To minimize this, please follow the following additional guidelines (the rest of this document also applies!).
```

Comment:
I think there's probably a better way to phrase this that sounds less like we expect working with external people to be frustrating. How about something like

```suggestion
In order to help, there are additional guidelines for external contributors to follow, in addition to other guidelines listed here.
Following these guidelines will greatly improve our ability and willingness to collaborate with you on your changes.
```

Reply:
>In order to help: 

To me, this isn't very genuine sounding without more explanation

>in addition to other guidelines listed here: 

This is redundant after the wording "additional guidelines"

I like Michael's original wording better because it gives more motivation. 

Maybe we can expand the motivation a bit?
```suggestion
To more effectively collaborate, please also follow the following additional guidelines. In our experience, following these guidelines avoids frustration and saves time.
```

Reply:
I think it's difficult to word this in a way that doesn't allude to the problems or difficulties. Jamie, to me your wording also reads as vaguely threatening - do this or else! How about this:

The Plutus team welcomes contributions from external contributors.                                                                                                                                                                                                                                                     
However, it can be difficult for the Plutus team to quickly review contributions from people where we don't have an existing relationship.
For that reason, we ask you to follow these additional guidelines (the rest of the document also applies!), which will make it easier for us to review your work, and therefore make the contributing process smoother for you.

Reply:
Yeah, I think that is the best way to phrase it. It's direct but it makes it seem like the goal is to improve collaboration rather than to avoid frustration